ros/src/tl_detector/tl_detector.py

```diff
@@ -113,13 +151,19 @@ def get_light_state(self, light):
             int: ID of traffic light color (specified in styx_msgs/TrafficLight)
 
         """
-        if(not self.has_image):
-            self.prev_light_loc = None
-            return False
+
+        # For testing, just return the light state provided by the simulator. Won't work real-world!!!
+        if self.use_tf_detection is False:
+            return light.state
+
+        if not self.has_image:
+            return self.last_state
 
         cv_image = self.bridge.imgmsg_to_cv2(self.camera_image, "bgr8")
+        cv_image = cv2.resize(cv_image, (self.preprocessing_width, self.preprocessing_height))
```

Comment:
Just double-checking, but hope the neural network was trained on BGR pixel format like OpenCV uses and not RGB like most other libraries seem to use? (might still get decent performance based on the location but colors would be wrong)

Reply:
I checked this ! :+1:  Thx